### PR TITLE
Update to match k8s 1.18.0

### PR DESCRIPTION
**Pull request by dholbach**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -300,5 +300,7 @@ If you have any questions about, feedback for or problems with `kured`:
 - Invite yourself to the <a href="https://slack.weave.works/" target="_blank">Weave Users Slack</a>.
 - Ask a question on the [#kured](https://weave-community.slack.com/messages/kured/) slack channel.
 - [File an issue](https://github.com/weaveworks/kured/issues/new).
+- Join us in [our monthly meeting](https://docs.google.com/document/d/1bsHTjHhqaaZ7yJnXF6W8c89UB_yn-OoSZEmDnIP34n8/edit#),
```
Comment (evrardjp) [inline README.md]: This should probably go into the other PR, #126 .

Comment (dholbach) [reply README.md]: Yes. Fixed by rebasing on current master.

**Comment by dholbach** [comment]
I followed https://github.com/weaveworks/kured/blob/master/DEVELOPMENT.md#release-testing to test this on 1.18.0.

**Comment by evrardjp** [comment]
I have this in my backlog for tomorrow. Sorry for the delay!

**Comment by dholbach** [comment]
> We might want to update the README too. Want to do it in another PR or this one?

Yes. Let's do one more PR to check all the relevant versions and see if there's anything else we need to update.

**Pull request merged**